### DISCUSSION TITLE
v1.11

### DISCUSSION THREAD

**Discussion by ocobles** [equinix/terraform-provider-equinix#280] [General]
## 1.11.1 (2 Dec, 2022)

BUG FIXES:

An error loading provider configuration, introduced in v1.11.0, was preventing using the provider without Fabric credentials https://github.com/equinix/terraform-provider-equinix/pull/283

## 1.11.0 (1 Dec, 2022)

FEATURES:

New Equinix Fabric v4 resources (**experimental**) in [#261](https://github.com/equinix/terraform-provider-equinix/pull/261) [#263](https://github.com/equinix/terraform-provider-equinix/pull/263) [#264](https://github.com/equinix/terraform-provider-equinix/pull/264)

- **New Resource:** `equinix_fabric_connection`
- **New Resource:** `equinix_fabric_service_profile`
- **New Data Source:** `equinix_fabric_connection`
- **New Data Source:** `equinix_fabric_port`
- **New Data Source:** `equinix_fabric_ports`
- **New Data Source:** `equinix_fabric_service_profile`
- **New Data Source:** `equinix_fabric_service_profiles`

BUG FIXES:

- `metal_connection.vlans` wasn't in sync with API [#271](https://github.com/equinix/terraform-provider-equinix/pull/271)
- Allow update of `equinix_metal_virtual_circuit.vlan_id` [#269](https://github.com/equinix/terraform-provider-equinix/pull/269)
- Allow set and update native vlan in `equinix_metal_port`, also for bond ports [#267](https://github.com/equinix/terraform-provider-equinix/pull/267)
- `equinix_metal_precreated_ip_block` and `equinix_metal_ip_block_ranges` will retrieve metro from facility if metro is null [#265](https://github.com/equinix/terraform-provider-equinix/pull/265)
- Remove optional flag for address fields in organization data source in [#261](https://github.com/equinix/terraform-provider-equinix/pull/261)

ENHANCEMENTS:

- `vlans` can be updated and removed for shared `equinix_metal_connection` [#273](https://github.com/equinix/terraform-provider-equinix/pull/273)
- Equinix terraform modules can now add prefix to the User-Agent by adding `module_name` field to the provider metadata [#255](https://github.com/equinix/terraform-provider-equinix/pull/255)

<hr /><em>This discussion was created from the release <a href='https://github.com/equinix/terraform-provider-equinix/releases/tag/v1.11.0'>v1.11.0</a>.</em>

**Comment by displague** [comment]
We're aware of a problem using the provider without Equinix Client ID and Client Secret values. 

While we address this issue (https://github.com/equinix/terraform-provider-equinix/issues/282) you can workaround the bug by defining `EQUINIX_API_TOKEN="-"` in the environment.

**Reply by displague** [reply]
Fixed in v1.11.1
